.github/workflows/gradle.yml

```diff
@@ -74,9 +74,11 @@ jobs:
       env:
         BROWSERSTACK_USER: ${{ secrets.BROWSERSTACK_USER }}
         BROWSERSTACK_KEY: ${{ secrets.BROWSERSTACK_KEY }}
+        APPLITOOLS_EXECUTE_KEY: ${{ secrets.APPLITOOLS_EXECUTE_KEY }}
+        APPLITOOLS_READ_KEY: ${{ secrets.APPLITOOLS_READ_KEY }}
       run: |
-        if [ -n $BROWSERSTACK_USER ] && [ -n $BROWSERSTACK_KEY ]; then
-            ./gradlew :vividus-tests:runStories -Pvividus.configuration.environments=system -Pvividus.configuration.suite=system -Pvividus.configuration.profile=web/desktop/chrome
+        if [ -n $BROWSERSTACK_USER ] && [ -n $BROWSERSTACK_KEY ] && [ -n $APPLITOOLS_READ_KEY ] && [ -n $APPLITOOLS_READ_KEY ]; then
```

Comment:
to make it more concise I would recommend to use [[ cond && cond ... ]] or [ cond -a cond ... ]